packages/frontend/src/components/dialogs/QrCode.tsx

```diff
@@ -228,8 +233,8 @@ export function QrCodeScanQrInner({
           openAlertDialog({
             message: error.message || error.toString(),
           })
+          processingQrCode.current = false
```

Comment:
Why has this been moved? The logic doesn't look right now.
And a copy-pasted version of this code is also in `ReceiveBackupDialog.tsx`, where it remains unchanged.

Reply:
if processQr is successfull it calls onDone which sets processingQrCode.current to false, but itf it fails it has to be set in the catch clause.

So both is correct, but I agree it is confusing to have it one time like this and one time like that

Reply:
Moved it back